### PR TITLE
feat - added homepage design

### PR DESCRIPTION
**Pull request by akash-1318**
Screenshot 2022-02-10 at 3 14 46 PM
Screenshot 2022-02-10 at 3 14 59 PM

Completed the homepage for E-commerce

### CODE REVIEW

File: pages/styles/common.css [hunk 1]
```diff
@@ -61,4 +61,16 @@ h6{
...
     text-decoration: none;
+}
+
+.header{
+    padding: 1rem 4rem;
+}
+
+.header__nav-right{
+    margin-left: 0px;
+}
```
Comment (tushargoyal812) [inline pages/styles/common.css:72]: is this mandatory?

**Review by tushargoyal812 — COMMENTED** [review]
looks good to merge just a suggestion use "beautify" extension in vs code to correct all the indentations.

File: pages/styles/common.css (hunk 1 above)
Comment (akash-1318) [reply pages/styles/common.css:72]: Yes actually i am using the header from component library i build in that margin-left is auto but here i don't want that.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,172 @@
...
+                        <p class="card__charc-name">New Arrival</p>
+                        <div>
+                            <p class="card__charc-series">Winter Collection</p>
+                            <p class="card__desc-text">Checkout our best winter Collection to stay hot in style this season.</p>
+                            </div>
+                      </div>
+                    </div>
+                </div>
+
+
```
Comment (arjunbharti) [inline index.html:147]: Remove extra spacing

**Review by arjunbharti — COMMENTED** [review]
Remove extra spacing in the code. Everything else is good

**Pull request merged**